File: rafael/operadores/GeraParcelas.java
class GeraParcelas {
	public static void main(String[] args) {	
		java.text.SimpleDateFormat sdf = new java.text.SimpleDateFormat("dd/MM/yyyy"); 
		
		java.util.Calendar p = new java.util.GregorianCalendar(2015 , 7 , 15) ;
		System.out.println(sdf.format(p.getTime()));

		p.add(java.util.Calendar.DAY_OF_MONTH, 30);
		System.out.println(sdf.format(p.getTime()));

		p.add(java.util.Calendar.DAY_OF_MONTH, 30);
		System.out.println(sdf.format(p.getTime()));

		p.add(java.util.Calendar.DAY_OF_MONTH, 30);
		System.out.println(sdf.format(p.getTime()));
	}
}

File: rafael/operadores/GeraParcelas.cs
class GeraParcelas 
{
	static void Main() 
	{	
		System.DateTime p = new System.DateTime(2015 , 8 , 15);
		System.Console.WriteLine(p.ToString("dd/MM/yyyy"));

		p = p.AddDays(30);
		System.Console.WriteLine(p.ToString("dd/MM/yyyy"));

		p = p.AddDays(30);
		System.Console.WriteLine(p.ToString("dd/MM/yyyy"));

		p = p.AddDays(30);
		System.Console.WriteLine(p.ToString("dd/MM/yyyy"));
	}
}
